.circleci/config.yml

```diff
@@ -124,6 +123,25 @@ jobs:
             if [[ -z "$version" ]] ; then version=${CIRCLE_SHA1:0:7}; fi
             aws s3 sync ~/artifacts s3://honeycomb-builds/honeycombio/refinery/$version/
 
+  publish_docker:
+    executor: linuxgo
+    steps:
+      - restore_cache:
+          keys:
+            googkleko
```

Comment:
Is the extra "k" in the key name meaningful? The save and restore keys match, so it works, just wondering if I'm missing meaning or if this was a typo.